internal/reconciler/tracepipeline/reconciler_test.go

```diff
@@ -905,14 +905,14 @@ func TestReconcile(t *testing.T) {
 				probeGatewayErr: &workloadstatus.PodIsPendingError{ContainerName: "foo", Reason: "OOMKilled"},
 				expectedStatus:  metav1.ConditionFalse,
 				expectedReason:  conditions.ReasonGatewayNotReady,
-				expectedMessage: "Pod is in the pending state because container: foo is not running due to: OOMKilled",
+				expectedMessage: "Pod is in the pending state because container: foo is not running due to: OOMKilled. Please check the container: foo logs.",
 			},
 			{
-				name:            "pod is craashbackloop",
+				name:            "pod is crashbackloop",
```

Comment:
nit: that state is not called crashbackloop, but CrashLoopBackOff

Reply:
Let's fix it separately